app/jobs/regular/export_csv_file.rb

```diff
@@ -103,7 +103,9 @@ def execute(args)
           if upload.persisted?
             user_export.update_columns(upload_id: upload.id)
           else
-            Rails.logger.warn("Failed to upload the file #{zip_filename}")
+            Rails.logger.warn(
+              "Failed to upload the file #{zip_filename}: #{upload.errors.full_messages}",
```

Comment:
Just following what we have done in other places: https://github.com/discourse/discourse/blob/499f5c25d286ad72b7416dd7440f6f4c735cfb29/app/jobs/regular/export_user_archive.rb#L190